src/components/send/transactionStatus/transactionStatus.js

```diff
@@ -178,7 +178,7 @@ class TransactionStatus extends React.Component {
                     className={`${styles.bookmarkDropdown}`}
                   >
                     <BookmarkDropdown
-                      delegate={recipientAccount.data.delegate || {}}
+                      delegate={(recipientAccount.data && recipientAccount.data.delegate) || {}}
```

Comment:
This change shouldn't be needed after you fixed the default value for `data` in `data: apis[key].defaultData || {},`